lib/openstax/exchange/client/client.rb

```diff
@@ -0,0 +1,67 @@
+module OpenStax
+  module Exchange
+    module Client
```

Comment:
I know this is not a Rails gem, but according to the Rails naming convention this file should be `lib/openstax/exchange/client.rb` (meaning you could combine this file and that file into one and it can live outside the client folder)